steering-and-technical-committee-charter.md

```diff
@@ -0,0 +1,226 @@
+
+The current membership of the STC is:
+
+| &nbsp;                                                         | Member           | Organization | Profile                                              | Term Start | Term End |
+| -------------------------------------------------------------- | ---------------- | ------------ | ---------------------------------------------------- | ---------- | --------
+| <img width="30px" src="https://github.com/yaron2.png"> | Yaron Schneider|   Microsoft     | [@yaron2](https://github.com/yaron2) | Bootstrap  | Nov 30th 2023     |
+| <img width="30px" src="https://github.com/msfussell.png">        | Mark Fussell      | Microsoft       | [@msfussell](https://github.com/msfussell)               | Bootstrap  | Nov 30th 2023     |
+| <img width="30px" src="https://github.com/beiwei30.png">      | Ian Luo  | Alibaba          | [@beiwei30](https://github.com/beiwei30)           | Bootstrap  | Nov 30th 2023     |
+| <img width="30px" src="https://github.com/cvictory.png">         | Victory Cao        | Alibaba  | [@cvictory](https://github.com/cvictory)                 | Bootstrap | Nov 30th 2023     |
+| <img width="30px" src="https://github.com/forrest-zhao.png">         | Forrest Zhao      | Intel       | [@forrest-zhao](https://github.com/forrest-zhao)                 | Bootstrap | Nov 30th 2023     |
+
+## Decision process
+
+The steering and technical committee desires to always reach consensus.
+
+Decisions requiring a vote include: issuing written governance policy, amending existing written governance policy, all spending, official responses to publicly raised issues, or any other decisions that at least half the members present decide require a vote.
+
+Decisions are made in meetings when a quorum of the members are present and may
+pass with at least half the members of the committee supporting it.
+
+Quorum is considered reached when at least half of the members are present.
+
+## Composition
+Seats on the steering and technical committee are held by an individual, not by their employer. This means that if an individual changes their employer, they retain their seat until the next election. 
+
+The steering and technical committee has 5 seats initially to bootstrap. These 5 seats are contributing seats, open to any project contributor. This is open to expand upto 11 seats for qualifying contributing members as the project expands. There is no timeframe for this expanded committee. 
+
+A maximum of two seats is allowed from any one company. For companies with two seats these are should be filled by one person focused on the vision, values, and mission of the project and another person who is focused on the technical decisions.
```

Comment:
Typo: `For companies with two seats these are should be filled`

Reply:
Updated